extension/src/reactor/index.ts

```diff
@@ -22,6 +22,166 @@ import {
 } from "./guards";
 import type { Status } from "./handlers";
 import { BreakpointMap, type Location } from "./locations";
+import path from "path";
+import { SessionType } from "../session/sessionType";
+import type { DebugpyProcessStoppedEvent, GDBProcessStoppedEvent, ProcessStoppedEvent } from "memviz-ui/dist/messages";
```

Comment:
We shouldn't import stuff from the `dist` folder, as that contains the raw JS files.